client/app/visualizations/counter/Renderer.jsx

```diff
@@ -44,15 +56,11 @@ export default function Renderer({ data, options, visualizationName }) {
     }
   }, [data, options, container]);
 
-  const {
-    showTrend,
-    trendPositive,
-    counterValue,
-    counterValueTooltip,
-    targetValue,
-    targetValueTooltip,
-    counterLabel,
-  } = getCounterData(data.rows, options, visualizationName);
+  const { counterLabel, showTrend, trendPositive, primaryValue, secondaryValue } = getCounterData(
```

Comment:
How about a `useMemo` to avoid recalculating on each re-render?

Reply:
Done :+1: